src/fortran/cwrapper.cpp

```diff
@@ -360,6 +366,16 @@ void NAME_MANGLE(heavy_thermal_diffusion_ratios) (double* const pk)
     p_mix->heavyThermalDiffusionRatios(pk);
 }
 
+//==============================================================================
+//void NAME_MANGLE(heavy_thermal_diffusion_ratios) (double* const pk)
+//{
+double NAME_MANGLE(electron_thermal_diffusionratios)()
```

Comment:
I would change "electron_thermal_diffusionratios" to "electron_thermal_diffusion_ratios" for consistency